releasetasks/templates/firefox/beetmove_image.yml.tmpl

```diff
@@ -0,0 +1,57 @@
+# keep first line
```

Comment:
Yeah, the first line is indented based on the position of the include macro in the main template, but applicable only for the first line.